gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -398,11 +398,11 @@ def GetCharacterHeader (pc):
 	Multi = GUICommon.HasMultiClassBits (pc)
 	Specific = "%d"%GemRB.GetPlayerStat (pc, IE_SPECIFIC)
```

Comment:
does anything depend on this being a string? Now we use a mix and you're changing it further, so it's a good chance to standardize.

Reply:
Glad you said so, because I thought it was odd. I searched for any instance where it was required to be a string, but couldn't find any, so changed it to a regular number.